addons/account/models/chart_template.py

```diff
@@ -47,7 +49,13 @@ def wrapper(*args, **kwargs):
                 # remove the template code argument as we already know it from the decorator
                 args, kwargs = args[:1], {}
             return func(*args, **kwargs)
-        return api.attrsetter('_l10n_template', (template, model))(wrapper)
+
+        # the module the function originates from is used for code translations
+        path = func.__globals__['__file__']
+        path_info = get_resource_from_path(path)
+        module = path_info[0] if path_info else 'account'
+
+        return api.attrsetter('_l10n_template', (template, model))(api.attrsetter('_module', module)(wrapper))
```

Comment:
I guess it's just to make it one line shorter, we kept the pattern of `odoo/api` but it might be more readable this way indeed :+1: 
My question was more about accessing `func.__globals__['__file__']`, do you see an issue there? It is used only to find the module with `get_python_translations`.